mat64/eigen.go

```diff
@@ -23,6 +29,90 @@ func symmetric(m *Dense) bool {
 	return true
 }
 
+// EigenSym is a type for creating and manipulating the Eigen decomposition of
```

Comment:
s/Eigen decomposition/eigendecomposition/ ? The terminology does not seem to be completely unified but eigendecomposition is maybe a bit more common.